Source/gmenu.cpp

```diff
@@ -177,53 +171,31 @@ void gmenu_draw()
 	}
 }
 
-void gmenu_draw_menu_item(TMenuItem *pItem, int a2)
+void gmenu_draw_menu_item(TMenuItem *pItem, int y)
 {
-	int v2;           // edi
-	TMenuItem *v3;    // ebx
-	unsigned int v4;  // eax
-	unsigned int v5;  // ebp
-	int v6;           // esi
-	unsigned int v7;  // ecx
-	unsigned int v8;  // eax
-	int v9;           // ecx
-	unsigned int v10; // ebp
-	int v11;          // esi
-	int v12;          // eax
-	int v13;          // edi
-	unsigned int v14; // [esp+10h] [ebp-4h]
-
-	v2 = a2;
-	v3 = pItem;
-	v4 = gmenu_get_lfont(pItem);
-	v5 = v4;
-	v14 = v4;
-	if (v3->dwFlags & 0x40000000) {
-		v6 = (v4 >> 1) + 80;
-		CelDecodeOnly(v6, v2 - 10, (BYTE *)optbar_cel, 1, 287);
-		v7 = (v3->dwFlags >> 12) & 0xFFF;
-		if (v7 < 2)
-			v7 = 2;
-		v8 = ((v3->dwFlags & 0xFFF) << 8) / v7;
-		v9 = (v5 >> 1) + 82;
-		v10 = v8;
-		gmenu_clear_buffer(v9, v2 - 12, v8 + 13, 28);
-		CelDecodeOnly(v6 + v10 + 2, v2 - 12, (BYTE *)option_cel, 1, 27);
-		v5 = v14;
+	int x;
+	DWORD w, xo;
+
+	w = gmenu_get_lfont(pItem);
+	if (pItem->dwFlags & 0x40000000) {
+		x = (w >> 1) + 80;
+		CelDecodeOnly(x, y - 10, optbar_cel, 1, 287);
+		xo = pItem->dwFlags >> 12 & 0xfff;
+		if (xo < 2) {
+			xo = 2;
+		}
+		xo = ((pItem->dwFlags & 0xfff) << 8) / xo;
+		gmenu_clear_buffer(x + 2, y + -12, xo + 13, 28);
+		CelDecodeOnly(xo + x + 2, y + -12, option_cel, 1, 27);
 	}
-	v11 = 384 - (v5 >> 1);
-	v12 = -((v3->dwFlags & 0x80000000) != 0);
-	_LOBYTE(v12) = v12 & 0xF1;
-	light_table_index = v12 + 15;
-	gmenu_print_text(384 - (v5 >> 1), v2, v3->pszStr);
-	if (v3 == sgpCurrItem) {
-		v13 = v2 + 1;
-		CelDecodeOnly(v11 - 54, v13, (BYTE *)PentSpin_cel, (unsigned char)PentSpin_frame, 48);
-		CelDecodeOnly(v11 + v5 + 4, v13, (BYTE *)PentSpin_cel, (unsigned char)PentSpin_frame, 48);
+	x = -(w >> 1) + 384;

```

Comment:
Maybe `384 - (w >> 1)`?

Reply:
Maybe style-wise? It doesn't change the output at least.